File: 프로그래머스/unrated/181879. 길이에 따른 연산/길이에 따른 연산.cpp
#include <string>
#include <vector>
#include <numeric>

using namespace std;

int solution(vector<int> num_list) {
    int answer = 1;
    if(num_list.size() >= 11) return accumulate(num_list.begin(),num_list.end(), 0);
    for(auto num : num_list){
        answer *= num;
    }
    return answer;
}

File: 프로그래머스/unrated/181879. 길이에 따른 연산/길이에 따른 연산.java
import java.util.*;

class Solution {
    public int solution(int[] num_list) {
        int answer = 1;
        if(num_list.length <= 10){
            for(int num: num_list){
                answer *= num;
            }
            return answer;
        }
        return Arrays.stream(num_list).sum();
    }
}